examples/js/lines/LineSegmentsGeometry.js

```diff
@@ -143,7 +143,7 @@ THREE.LineSegmentsGeometry.prototype = Object.assign( Object.create( THREE.Insta
 
 		} else if ( geometry.isBufferGeometry ) {
 
-			this.setPositions( geometry.position.array ); // assumes non-indexed
+			this.setPositions( geometry.attributes.position.array ); // assumes non-indexed
```

Comment:
The method name also has a spelling error. It should be `fromLineSegments()`.

Would you like to fix that, too?

Reply:
No problem. I'll check it out and try to fix it.:smile:

Reply:
Can you also fix the `d.ts` file? 😇 

https://github.com/mrdoob/three.js/blob/df2b31f86dbc82b4d852896c3f85f1a15f4b9f42/examples/jsm/lines/LineSegmentsGeometry.d.ts#L19

Reply:
I am so sorry ,I forgot it . 😄